### PR TITLE
Set Powershell encoding to UTF-8

### PR DESCRIPTION
**Pull request by dhaavi**
Also:

- Improve name cleaner
- Move debug util to separate pkg to avoid import loop

### CODE REVIEW

File: api/endpoints_debug.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func printStack(_ *Request) (msg string, err error) {
 // debugInfo returns the debugging information for support requests.
 func debugInfo(ar *Request) (data []byte, err error) {
 	// Create debug information helper.
-	di := new(osdetail.DebugInfo)
+	di := new(debug.DebugInfo)
 	di.Style = ar.Request.URL.Query().Get("style")
```
Comment (ppacher) [inline api/endpoints_debug.go:69]: I'd rename `debug.DebugInfo` to  `debug.Info` (go-vet should complain about it)

File: utils/osdetail/binmeta.go [hunk 2]
```diff
@@ -67,6 +68,11 @@ func GenerateBinaryNameFromPath(path string) string {
 func cleanFileDescription(fileDescr string) string {
 	fields := strings.Fields(fileDescr)
 
+	// Clean out and `"` and `'`.
+	for i := range fields {
+		fields[i] = cleanName.ReplaceAllString(fields[i], "")
+	}
```
Comment (ppacher) [inline utils/osdetail/binmeta.go:74]: Using [`strings.Replacer`](https://golang.org/pkg/strings/#Replacer) is probably more efficient than a regex search.

File: utils/osdetail/powershell_windows.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ func runPowershellCmd(script string) (output string, err error) {
 		"powershell.exe",
 		"-NoProfile",
 		"-NonInteractive",
-		script,
+		"[System.Console]::OutputEncoding = [System.Text.Encoding]::UTF8\n"+script,
 	)
```
Comment (ppacher) [inline utils/osdetail/powershell_windows.go:17]: :1st_place_medal:

**Review by ppacher — DISMISSED** [review]

File: api/endpoints_debug.go (hunk 1 above)
Comment (dhaavi) [reply api/endpoints_debug.go:69]: I figured, but wasn't up for it at the moment.

Comment (dhaavi) [reply api/endpoints_debug.go:69]: Will fix ;)

File: utils/osdetail/binmeta.go (hunk 2 above)
Comment (dhaavi) [reply utils/osdetail/binmeta.go:74]: Nice! Somehow was not aware of that.

**Review by ppacher — APPROVED** [review]

**Pull request merged**